packages/orchestration/src/facade.js

```diff
@@ -1,20 +1,144 @@
 /** @file Orchestration service */
 
+import { E } from '@endo/far';
+
 /**
  * @import {Zone} from '@agoric/zone';
  * @import {TimerService} from '@agoric/time';
+ * @import {LocalChain} from '@agoric/vats/src/localchain.js';
+ * @import {ERef} from '@endo/far';
  * @import {OrchestrationService} from './service.js';
- * @import {Orchestrator} from './types.js';
+ * @import {Chain, ChainInfo, OrchestrationAccount, Orchestrator} from './types.js';
+ */
+
+/** @type {any} */
+const anyVal = null;
+
+/**
+ * @param {ERef<LocalChain>} localchain
+ * @returns {Chain}
  */
+const makeLocalChainFacade = localchain => {
+  return {
+    /** @returns {Promise<ChainInfo>} */
+    async getChainInfo() {
+      return {
+        allegedName: 'agoric',
+        allowedMessages: [],
+        allowedQueries: [],
+        chainId: 'agoric-3',
+        ibcConnectionInfo: anyVal,
+        ibcHooksEnabled: true,
+        icaEnabled: true,
+        icqEnabled: true,
+        pfmEnabled: true,
+      };
+    },
+    async makeAccount() {
+      const account = await E(localchain).makeAccount();
+
+      return {
+        deposit(payment) {
+          console.log('deposit got', payment);
+          // XXX yet again tripped up on remote methods looking local statically
+          return E(account).deposit(payment);
+        },
+        transferSteps(amount, msg) {
+          console.log('transferSteps got', amount, msg);
+          return Promise.resolve();
+        },
+      };
+    },
+  };
+};
+
+/**
+ * @template {string} C
+ * @param {C} name
+ * @returns {Chain}
+ */
+const makeRemoteChainFacade = name => {
+  const chainInfo = {
+    allegedName: name,
+    chainId: 'fixme',
+  };
+  return {
+    /** @returns {Promise<ChainInfo>} */
+    getChainInfo: async () => anyVal,
+    /** @returns {Promise<OrchestrationAccount<C>>} */
+    makeAccount: async () => {
+      console.log('makeAccount for', name);
+      return {
+        delegate(validator, amount) {
+          console.log('delegate got', validator, amount);
+          return Promise.resolve();
+        },
+        deposit(payment) {
+          console.log('deposit got', payment);
+          return Promise.resolve();
+        },
+        getAddress() {
+          return {
+            chainId: chainInfo.chainId,
+            address: 'an address!',
+            addressEncoding: 'bech32',
+          };
+        },
+        getBalance(denom) {
+          console.error('getBalance not yet implemented');
+          return { brand: null, value: 0n };
```

Comment:
```suggestion
          return Promise.resolve({ denom: 'fixme', value: 0n });
```
I'm not sure how to resolve all of the type errors for `makeRemoteChainFacade().makeAccount`, but this seems to be one